keycloak/data/havendev-realm.json

```diff
@@ -1,1598 +1,1778 @@
-      "salt" : "6u4mycWng8AhcPz2C+erhg==",
-      "hashIterations" : 20000,
-      "counter" : 0,
-      "algorithm" : "pbkdf2-sha256",
-      "digits" : 0,
-      "period" : 0,
-      "createdDate" : 1503871865370,
-      "config" : { }
-    } ],
-    "disableableCredentialTypes" : [ "password" ],
-    "requiredActions" : [ ],
-    "realmRoles" : [ "uma_authorization", "offline_access" ],
-    "clientRoles" : {
-      "account" : [ "manage-account", "view-profile" ]
+    {
+      "id": "63edc28f-95e6-4250-b784-67255c2d9001",
+      "name": "org-none",
+      "path": "/org-none",
+      "attributes": {},
+      "realmRoles": [],
+      "clientRoles": {},
+      "subGroups": []
     },
-    "groups" : [ ]
-  }, {
-    "id" : "90920d91-3090-4b4a-ae2a-2377cfa06ecd",
-    "createdTimestamp" : 1492023544974,
-    "username" : "user1@havengrc.com",
-    "enabled" : true,
-    "totp" : false,
-    "emailVerified" : true,
-    "firstName" : "User1",
-    "lastName" : "One",
-    "email" : "user1@havengrc.com",
-    "attributes" : {
-      "role" : [ "member" ]
+    {
+      "id": "8e1acd95-1ce2-4c16-963c-bbb1a25ca5ca",
+      "name": "staff",
+      "path": "/staff",
+      "attributes": {},
+      "realmRoles": [],
+      "clientRoles": {},
+      "subGroups": []
     },
-    "credentials" : [ {
-      "type" : "password",
-      "hashedSaltedValue" : "MsHzlmMXOxNq6lucfqadNs2UyfNsmEOngrHOB0hINLSGMD2mZvsvVfjtbVmRP03XqeelY5sXxc/AAf7qfeN4Jw==",
-      "salt" : "FmeRCCF1AtEsy+RekV6+Cw==",
-      "hashIterations" : 20000,
-      "counter" : 0,
-      "algorithm" : "pbkdf2-sha256",
-      "digits" : 0,
-      "period" : 0,
-      "createdDate" : 1505441740106,
-      "config" : { }
-    } ],
-    "disableableCredentialTypes" : [ "password" ],
-    "requiredActions" : [ ],
-    "realmRoles" : [ "uma_authorization", "member", "offline_access" ],
-    "clientRoles" : {
-      "havendev" : [ "member" ],
-      "account" : [ "view-profile", "manage-account" ]
+    {
+      "id": "8f3b3184-d05e-477c-acd8-a44d46544b94",
+      "name": "vendor",
+      "path": "/vendor",
+      "attributes": {},
+      "realmRoles": [],
+      "clientRoles": {},
+      "subGroups": []
+    }
+  ],
+  "defaultRoles": [
+    "uma_authorization",
+    "member",
+    "offline_access"
+  ],
+  "defaultGroups": [
+    "/org-none"
+  ],
+  "requiredCredentials": [
+    "password"
+  ],
+  "passwordPolicy": "hashIterations(20000)",
+  "otpPolicyType": "totp",
+  "otpPolicyAlgorithm": "HmacSHA1",
+  "otpPolicyInitialCounter": 0,
+  "otpPolicyDigits": 6,
+  "otpPolicyLookAheadWindow": 1,
+  "otpPolicyPeriod": 30,
+  "otpSupportedApplications": [
+    "FreeOTP",
+    "Google Authenticator"
+  ],
+  "clients": [
+    {
+      "id": "f292fcd9-4b25-4d58-bddc-9d29a72d7a59",
+      "clientId": "admin-cli",
+      "name": "${client_admin-cli}",
+      "surrogateAuthRequired": false,
+      "enabled": true,
+      "clientAuthenticatorType": "client-secret",
+      "secret": "**********",
```

Comment:
I'm not sure what to think about this line, previously it was "6cae6b16-4de1-4aa7-88b6-7dd0efa81b17"